src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -19,9 +20,12 @@ function HeatmapVis(props: Props): JSX.Element {
 
   const domain = findDomain(values);
   const [colorMap, setColorMap] = useState<ColorMap>('Magma');
+  const [logScale, setLogScale] = useState<boolean>(false);
```

Comment:
I use a second State (the first being for the name of the colormap) to track the value of the logScale boolean and trigger rendering. I wonder if this is the best way to do it.

Reply:
I'm working on implementing a common heatmap state. 👍